app.js

```diff
@@ -31,6 +33,22 @@ client.on('messageDelete', async msg => {
   msg.channel.send(`${author} deleted message: ${message}`);
 });
 
+client.on('presenceUpdate', (oldPresence, newPresence) => {
+  if(oldPresence === undefined) {
+    return;
+  }
+  if(oldPresence.status === newPresence.status) {
+    return;
+  } 
+  if(oldPresence.status !== 'offline' && oldPresence.status !== 'idle') {
+    return;
+  }
+  if(newPresence.status !== 'online') {
+    return;
+  }
+  var user = client.users.cache.find(user => user.id === newPresence.userID)
+  client.channels.cache.get('266042248548974603').send(user.username + ' is online!');
+});
```

Comment:
Can you change to const instead of var

And I think user already exists on the presence object, I don't think you need to find it. You should be able to do just `.send(newPresence.user.username + ' is online!');`